prime-router/src/main/kotlin/fhirengine/translation/hl7/FhirTransformer.kt

```diff
@@ -178,22 +164,42 @@ class FhirTransformer(
         bundle: Bundle,
         focusResource: Base
     ) {
-        if (bundleProperty == null) return
+        if (bundleProperty.isNullOrBlank()) {
+            logger.warn("bundleProperty was not set.")
+            return
+        }
 
         val pathParts = bundleProperty.split(".")
+        if (pathParts.size < 2) {
+            logger.warn("Expected at least 2 parts in bundle property '$bundleProperty'.")
```

Comment:
Not sure this limitation is needed. Are you sure?

Reply:
Yes, without it we can't use `setProperty` since it expects a property name and doesn't use fhir path.